doc/neps/nep-0052-python-api-cleanup.rst

```diff
@@ -181,9 +230,20 @@ Details TBD, something like::
     numpy.doc
     numpy.matlib
     numpy.version
-    
+
     # To clean out or somehow deal with: everything in `numpy.lib`
 
+.. S: Are you thinking that even math.* will disappear out of the name
+   mainspace? That's quite a big change.  I like the principle you
+   proposed on Sebastian's PR above: one function, one home.
+
+.. S: Will we preserve `np.lib` as per the above discussion?
```

Comment:
I think it would be a break to move it, for `stride_tricks` in particular probably. That said, there's very little of use in any of the other subsubmodules .... so maybe?